master/internal/api_ntsc_intg_test.go

```diff
@@ -77,92 +95,139 @@ func TestTasksCountAuthZ(t *testing.T) {
 
 func TestCanGetNTSC(t *testing.T) {
 	api, authz, curUser, ctx := setupNTSCAuthzTest(t)
+	var err error
 
-	nbsActor := actor.AddrFromString("/notebooks")
-	nbID := setupMockNBActor(t, api.m)
 	// check permission errors are returned with not found status and follow the same pattern.
 	authz.On("CanGetNSC", mock.Anything, curUser, mock.Anything, mock.Anything).Return(
 		false, nil,
-	).Once()
+	).Times(2)
```

Comment:
question: whats the times 2 for?

Reply:
asserts that it can only be called up to two times